### DISCUSSION TITLE
Stream read and write rows to file

### DISCUSSION THREAD

**Discussion by multiaki** [exceljs/exceljs#1722] [Q&A]
I have 800MB excel files. I need to update columns and save the file. 
Is there a way to stream chunk of a file, modify and save to another file. 
The problem I see with current implementation is that file gets corrupted on save.

Please help with any suggestions. 
Thanks

**Comment by legopin** [comment]
The docs for streaming excel is here
https://github.com/exceljs/exceljs#streaming-xlsx

If you could have some code and sample showing the file corruption others will be able to help you better
